### DISCUSSION TITLE
Tooltip gets hidden in ModalBody

### DISCUSSION THREAD

**Discussion by spatil32** [twilio-labs/paste#714] [Q&A] [answered]
Hi, we have a requirement to show a text with Tooltip in ModalBody. The tooltip gets hidden in modal body because of low z-index value.
![Screen Shot 2020-09-01 at 4 01 06 PM](https://user-images.githubusercontent.com/16721217/91914904-0b65cc80-ec6e-11ea-9738-7b3dd36603f0.png)

**Comment by kate-porter** [comment]
Hey @spatil32 - thanks for pointing this out. I've just created a [bug ticket](https://issues.corp.twilio.com/browse/DSYS-2246) for it. What kind of timeline are you working with?

**Reply by spatil32** [reply]
Hey @katieporter, thanks for the quick response. The feature I am working on is the Signal deliverable, so if we could fix the bug in next couple of days, it would be a great help. Meanwhile, per PM's feedback, I am just showing the duration as plain text.

**Reply by richbachman** [reply]
Hi @spatil32. The team is actively discussing the solution, and we will prioritize in an upcoming sprint. Basically modals are a bit tricky and we want to make sure we fix the issue correctly. Thanks for coming up with a work around. We'll update you as soon as we have a fix in place.

**Comment by richbachman** [comment, marked as answer]
Hi @spatil32. We just merged in a fix for the popovers and tooltips in modals z-index issue. If you update to the latest version of core, you should see that fix. Let us know if you run into any other issues.

**Reply by spatil32** [reply]
@richbachman Thank you for adding the fix. 👍  I can confirm that tooltip is now working in the Modal. Just a small heads up, updating the version might have introduced a prop type warning as in the screenshot below? (Nothing major or breaking for now though). 

<img width="1134" alt="Screen Shot 2020-09-15 at 2 17 51 PM" src="https://user-images.githubusercontent.com/16721217/93266183-8d6def00-f75e-11ea-9566-4fccce067431.png">

**Reply by richbachman** [reply]
I think you may just need to update the design-tokens package as well. You can see the latest package versions here: https://paste.twilio.design/getting-started/engineering#install-paste-system-packages

We're working on a better way to keep the Paste packages updated. More to come soon.
